src/gameplay_v2/roles/components/seePercival.ts

```diff
@@ -0,0 +1,6 @@
+import Component from './component';
+
+export class SeePercivalC extends Component {
+  name = 'SeeSpies';
+  data = {};
+}
```

Comment:
`SeePercivalInfoC`. We're exposing what Percival should see. Not that we're seeing Percival!